tests/handlers/botas/test_botas_shipped_advisory.py

```diff
@@ -98,11 +99,121 @@ def test_handle_isnt_allowed_by_internal_policy(self):
         self.assertTrue(db_event.state_reason.startswith(
             "This image rebuild is not allowed by internal policy."))
 
-    @patch.object(conf, 'dry_run', new=True)
-    @patch.object(conf, 'handler_build_allowlist', new={
-        'HandleBotasAdvisory': {
-            'image': {
-                'advisory_name': 'RHBA-2020'
+    def test_handle(self):
+        event = BotasErrataShippedEvent("test_msg_id", self.botas_advisory)
+        self.handler.allow_build = MagicMock(return_value=True)
+        self.handler._create_original_to_rebuilt_nvrs_map = \
+            MagicMock(return_value={"original_1": "some_name-1-12345",
+                                    "original_2": "some_name_2-2-2"})
+        nvr_to_digest = {
+            "original_1": "original_1_digest",
+            "some_name-1-12345": "some_name-1-12345_digest",
+            "original_2": "original_2_digest",
+            "some_name_2-2-2": "some_name_2-2-2_digest",
+        }
+        bundles = [{"bundle_path_digest": "original_1_digest"},
+                   {"bundle_path_digest": "some_name-1-12345_digest"},
+                   {"bundle_path_digest": "original_2_digest"},
+                   {"bundle_path_digest": "some_name_2-2-2_digest"}]
+        related_bundles = {"original_1_digest": [{"bundle_path_digest": "related_1_digest"}],
```

Comment:
`related_` is confusing here since it's not a related image, this is the digest of the bundle.